packages/core/src/RenderingEngine/helpers/setDefaultVolumeVOI.ts

```diff
@@ -175,7 +175,13 @@ async function getVOIFromMinMax(
   // instead. For the first scenario, we use the arrayBuffer of the volume to get the correct
   // slice for the imageScalarData, and for the second scenario we use the getPixelData
   // on the Cornerstone IImage object to get the pixel data.
-  const image = await loadAndCacheImage(imageId, options);
+  let image;
+  // Note: we don't want to use the derived or generated images for setting the
```

Comment:
```suggestion
  // Note: we don't want to use the derived or generated images for setting the
```
Couple of thoughts on this - one idea is to create a utility function to do the test on the imageId portion, which at least abstracts it out of this class.  Another idea is to not check on the imageId, but to check on the image object itself.  That means that loadAndCacheImage would have to response with a resolved promise, but that should probably already be true, so doing it that way means you can add a field on the image response object.

Reply:
Actually, the cache itself has a referenced image/volume, so you can use that to determine if it is derived/generated?